src/erl/tests.rs

```diff
@@ -1206,13 +1091,10 @@ x() ->
         r#"-module(the_app).
 -compile(no_auto_import).
 
--export(['receive'/0, 'catch'/1]).
-
-'receive'() ->
-    'try':'and'().
+-export(['catch'/1]).
 
 'catch'(X) ->
-    'receive'().
+    try:'and'().
```

Comment:
Is this valid Erlang? `try` is used for try/catch

Reply:
I would assume not. It looks like none of the erlang printed in the `call` function escapes the module names at the moment. Should they all be wrapped in `atom()`?

Reply:
I'll make an issue